lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -11,10 +11,9 @@
 
   def self.instances
     PuppetX::PuppetLabs::ScheduledTask::V1Adapter.tasks.collect do |job_file|
```

Comment:
Can probably rename the iterator here as task as job_file has no meaning in the V2 land